raiden/tests/unit/test_tokennetwork.py

```diff
@@ -1126,5 +1141,163 @@ def test_routing_priority(
         to_address=address3,
         amount=1,
         previous_address=None,
+        config={},
     )
     assert routes[0].node_address == address2
+
+
+def test_routing_mocked_pfs(
```

Comment:
There should also be (a) test(s) for the unhappy case(s) (re. error handling etc. from above)